### PR TITLE
Add Check to textutils.tabulate/pagedTabulate

### PR DESCRIPTION
**Pull request by JakobDev**

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua [hunk 1]
```diff
@@ -122,6 +122,9 @@ local function tabulateCommon( bPaged, ... )
     for n, t in ipairs( tAll ) do
         if type(t) == "table" then
             for n, sItem in pairs(t) do
+                if type( sItem ) ~= "number" and type( sItem ) ~= "string" then
+                    error( "textutils.tabulate/pagedTabulate only allow strings and numbers in the table", 3 ) 
+                end
```
Comment (dan200) [inline src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua]: Please only allow strings here, and format the error message more standardly.

Comment (BombBloke) [reply src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua]: Why only strings, all of a sudden? What happened to "no breaking changes"?

Comment (Wilma456) [reply src/main/resources/assets/computercraft/lua/rom/apis/textutils.lua]: What should the message look like? e.g bad argument #Position.Position in Table (expected string, got foo)?

**Comment by Wilma456** [comment]
@dan200 Changes are now done

**Pull request closed without merging**